CMakeLists.txt

```diff
@@ -621,30 +621,26 @@ endif()
 #
 # this gets the libraries needed by TARGET in "-libx -liby ..." form
 #
-function(get_link_libraries OUT TARGET)
```

Comment:
need to check this one

Reply:
Well, not per default, but there might be: https://www.freedesktop.org/wiki/Software/pkg-config/

Reply:
> need to check this one

This is fine for me, ADDITIONAL_LIBS correctly contains all the required libs, and i makes the code easier to read.